README.md

```diff
@@ -1,5 +1,7 @@
 #SocketPlane
 
+[![Circle CI](https://circleci.com/gh/socketplane/socketplane/tree/master.svg?style=svg)](https://circleci.com/gh/socketplane/socketplane/tree/master) [![Coverage Status](https://img.shields.io/coveralls/dave-tucker/socketplane.svg)](https://coveralls.io/r/dave-tucker/socketplane)
+
```

Comment:
There is nothing wrong per-se. But just checking if we need to do anything to avoid using dave-tucker account in coveralls and use socketplane account ?